CoronaTracker.Core.Tests.Unit/Services/Processings/ExternalCountries/ExternalCountryProcessingServiceTests.Logic.RetrieveAll.cs

```diff
@@ -0,0 +1,44 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using CoronaTracker.Core.Models.ExternalCountries;
+using FluentAssertions;
+using Force.DeepCloner;
+using Moq;
+using Xunit;
+
+namespace CoronaTracker.Core.Tests.Unit.Services.Processings.ExternalCountries
+{
+    public partial class ExternalCountryProcessingServiceTests
+    {
+        [Fact]
+        public async Task ShouldRetrieveAllExternalCountriesAsync()
+        {
+            // given
+            List<ExternalCountry> randomCountries = CreateRandomExternalCountries();
+            List<ExternalCountry> externalCountries = randomCountries;
+            List<ExternalCountry> expectedExternalCountries = externalCountries.DeepClone();
+            this.externalCountryServiceMock.Setup(service =>
```

Comment:
This mock setup has two issues:
1. put a new line between multi-liners and one-liners.
2. Fix the tab on the return